src/arch/Cargo.toml

```diff
@@ -11,13 +11,11 @@ libc = ">=0.2.39"
 vm-memory = { path = "../vm-memory" }
 versionize = ">=0.1.6"
 versionize_derive = ">=0.1.3"
+vm-fdt = "0.1.0"
 
 arch_gen = { path = "../arch_gen" }
 logger = { path = "../logger" }
 utils = { path = "../utils" }
 
 [dev-dependencies]
-device_tree = ">=1.1.0"
-
-[target.'cfg(target_arch="aarch64")'.dependencies]
-libfdt-bindings = { path = "../libfdt-bindings" }
```

Comment:
Nit: This should be part of the 3rd commit. Also should we add a new line at the end of the file ?